docs/pl/docs/index.md

```diff
@@ -24,7 +24,7 @@
 
 ---
 
-FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący na standardowym typowaniu Pythona.
+FastAPI to nowoczesny, wydajny framework webowy do budowania API z użyciem Pythona 3.6+ bazujący się na standardowym typowaniu Pythona.
```

Comment:
Huh, I ran my suggestion by a native speaker and she says it should be `framework ... bazujący na ... typowaniu`. Without `się`.

So sorry I led you astray 🙈

Reply:
Don't worry, mistakes happen.

Reply:
Reverted